core/domain/email_manager_test.py

```diff
@@ -2645,3 +2646,365 @@ def test_correct_moderator_action_email_sent(self):
+
+    def test_send_assigned_question_reviewer_email(self):
+        expected_email_subject = (
+            'You have been invited to review Oppia questions')
+        expected_email_html_body = (
+            'Hi question,<br><br>'
+            'This is to let you know that the Oppia team has added you as a '
+            'reviewer for questions. This allows you to review question '
+            'suggestions made by contributors.<br><br>'
+            'You can check the question suggestions waiting for review in the '
+            '<a href="https://www.oppia.org/community_dashboard">'
+            'Community Dashboard</a>.<br><br>'
+            'Thanks, and happy contributing!<br><br>'
+            'Best wishes,<br>'
+            'The Oppia Community')
+
+        with self.can_send_emails_ctx:
+            email_manager.send_email_to_new_community_reviewer(
+                self.question_reviewer_id,
+                constants.REVIEW_CATEGORY_QUESTION, language_code='hi')
+
+            # Make sure correct email is sent.
+            messages = self.mail_stub.get_sent_messages(
+                to=self.QUESTION_REVIEWER_EMAIL)
+            self.assertEqual(len(messages), 1)
+            self.assertEqual(
+                messages[0].html.decode(), expected_email_html_body)
+
+            # Make sure correct email model is stored.
+            all_models = email_models.SentEmailModel.get_all().fetch()
+            sent_email_model = all_models[0]
+            self.assertEqual(
+                sent_email_model.subject, expected_email_subject)
+            self.assertEqual(
+                sent_email_model.recipient_id, self.question_reviewer_id)
+            self.assertEqual(
+                sent_email_model.recipient_email, self.QUESTION_REVIEWER_EMAIL)
+            self.assertEqual(
+                sent_email_model.sender_id, feconf.SYSTEM_COMMITTER_ID)
+            self.assertEqual(
+                sent_email_model.sender_email,
+                'Site Admin <%s>' % feconf.NOREPLY_EMAIL_ADDRESS)
+            self.assertEqual(
+                sent_email_model.intent, feconf.EMAIL_INTENT_ONBOARD_REVIEWER)
+
+    def test_email_is_not_sent_can_send_emails_is_false(self):
+        with self.can_not_send_emails_ctx:
+            email_manager.send_email_to_removed_community_reviewer(
+                self.translation_reviewer_id,
+                constants.REVIEW_CATEGORY_TRANSLATION, language_code='hi')
+
+        messages = self.mail_stub.get_sent_messages(
+            to=self.TRANSLATION_REVIEWER_EMAIL)
+        self.assertEqual(len(messages), 0)
+
+    def test_remove_translation_reviewer_email_for_invalid_review_category(
+            self):
+        with self.assertRaisesRegexp(Exception, 'Invalid review_category'):
+            email_manager.send_email_to_removed_community_reviewer(
+                self.translation_reviewer_id, 'invalid_category')
+
+    def test_all_category_keys_present_in_removed_reviewer_email_data_constant(
```

Comment:
See both comments above re the other constant. Similar checks should be applied here.

Reply:
Done!